### PR TITLE
Update github actions to no longer create PRs

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
* No longer create PRs during release actions.
* Update workflow to instead have some text where the PR creation was
* Update release instructions to do this manually

# Why
Due to a security issue we can no longer have github actions put up PRs:
image
https://aka.ms/opensource/actions-changes-230217

# How Tested
n/a

### CODE REVIEW

**Review by dmceachernmsft — APPROVED** [review]

**Comment by edwardlee-msft** [comment]
T.T

File: docs/releases/creating-a-release.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Use the [create-prerelease-branch](https://github.com/Azure/communication-ui-lib
 This section describes what the workflow above does. Understanding the workflow actions is useful in case the workflow fails.
 
 1. Create a `prerelease/<release-tag>` branch, bump the package version for `@azure/communication-react` as appropriate and collect all change files into a changelog.
-1. Creates another branch for manually summarizing the changes collected in the changelog and opens a Pull Request into the `prerelease/<release-tag>`. **Merge this PR before going to step 1.2**.
+1. Creates another branch for manually summarizing the changes collected in the changelog and ~~opens a Pull Request into the `prerelease/<release-tag>`~~ per [security policy](https://aka.ms/opensource/actions-changes-230217) you must open a PR manually of this new branch (`groom-changelog/<release-tag>`) into the prerelease branch (`prerelease/<release-tag>`). **Merge this PR before going to step 1.2**.
```
Comment (edwardlee-msft) [inline docs/releases/creating-a-release.md:40]: Should this be labeled as 2.?

**Review by edwardlee-msft — APPROVED** [review]

File: docs/releases/creating-a-release.md (hunk 1 above)
Comment (JamesBurnside) [reply docs/releases/creating-a-release.md:40]: Updated for clarity on step no.

**Comment by JamesBurnside** [comment]
> T.T

My feeling is we should move this to ADO, but can be a good retro item

**Pull request merged**